core/hooks/substrate/useAccounts.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { DeriveBalancesAccount } from '@polkadot/api-derive/types';
+import { useApi } from '../useApi';
+
+interface AccountWithBalance extends DeriveBalancesAccount {
+  address: string;
+}
+
+export const useAccounts = (): AccountWithBalance[] | undefined => {
+  const [accounts, setAccounts] = useState<AccountWithBalance[]>();
```

Comment:
This hook is a duplicate of existing features. We have `accounts` in `useExtension()`. We also have the `useBalance()` that will return balance information.

Reply:
Maybe something like `useBalances` would make sense, but we need to specify which chain you are interested in. This feature is out of scope for the v `1.0.0` release.